preseed/finish.erb

```diff
@@ -26,17 +26,9 @@ oses:
 /bin/sed -i "s/^<%= @host.ip %>.*/<%= @host.ip %>\t<%= @host.shortname %>.<%= @host.domain %>\t<%= @host.shortname %>/g" /etc/hosts
 /bin/echo <%= @host.shortname %> > /etc/hostname
 <% end -%>
+
 <% if puppet_enabled %>
-cat > /etc/puppet/puppet.conf << EOF
-<%= snippet 'puppet.conf' %>
-EOF
-if [ -f "/etc/default/puppet" ]
-then
-/bin/sed -i 's/^START=no/START=yes/' /etc/default/puppet
-fi
-/bin/touch /etc/puppet/namespaceauth.conf
```

Comment:
Yes, the only information I could find was that the file is required to be present (but not evaluated) for 2.7.x because of some bug, but I think it's time to bid Puppet 2.7 farewell.